examples/default/src/components/PlatformListTie.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { PropsWithChildren } from 'react';
+
+import { Box, HStack, Pressable, Text } from 'native-base';
+import { Platform } from 'react-native';
+
+interface PlatformListTileProps extends PropsWithChildren {
+  title: string;
+  onPress?: () => void;
+  platform?: 'ios' | 'android';
+}
+
+export const PlatformListTile: React.FC<PlatformListTileProps> = ({
```

Comment:
I think I missed this when creating the branch, but file name is `PlatformListTie` and Class is `PlatformListTile`, there's a missing L in Tile. 🙈 🙏

Reply:
fixed,, done